lnwallet/fee_estimator.go

```diff
@@ -202,23 +222,21 @@ func (b *BtcdFeeEstimator) fetchEstimatePerVSize(
 		return 0, err
 	}
 
-	// The value returned is expressed in fees per KB, while we want
-	// fee-per-byte, so we'll divide by 1000 to map to satoshis-per-byte
-	// before returning the estimate.
-	satPerByte := SatPerVByte(satPerKB / 1000)
-
-	// Before proceeding, we'll make sure that this fee rate respects the
-	// minimum relay fee set on the backend node.
-	if satPerByte < b.minFeeRate {
-		walletLog.Debugf("Using backend node's minimum relay fee rate "+
-			"of %v sat/vbyte", b.minFeeRate)
-		satPerByte = b.minFeeRate
+	// Since we use fee rates in sat/kw internally, we'll convert the
+	// estimated fee rate from its sat/kb representation to sat/kw.
+	satPerKw := SatPerVByte(satPerKB / 1000).FeePerKWeight()
```

Comment:
should probably create a `SatPeKB->SatPerKw` method to do this conversion without precision loss.

Reply:
Actually, I suggest renaming `SatPerVByte`->`SatPerKVByte`, then we can do the conversion as is done here (removing the division by 1000 ofc)

Reply:
Fixed.